web/src/elements/ak-locale-context/ak-locale-context.ts

```diff
@@ -59,8 +63,11 @@ export class LocaleContext extends LitElement {
 
     connectedCallback() {
         super.connectedCallback();
-        const localeRequest = autoDetectLanguage(this.locale);
-        this.updateLocale(localeRequest);
+        new CoreApi(DEFAULT_CONFIG)
+            .coreUsersMeRetrieve()
+            .then((user) => (this.userLocale = user?.user?.settings?.locale ?? ""))
+            .catch(() => {});
```

Comment:
Can't use `me()` here, it'll cause an infinite loop since it'll 403 when not logged in and *force* a redirect to the login page.  Which will then loop.  Doing it by hand allows us to avoid that.  We get the `EVENT_REFRESH` event anyway when we transition from the Flow to the Admin or User interfaces.

Reply:
Overall I think it would probably be a better pattern to not have `me()` do a redirect and just add a mixin that redirects to login if not authenticated, but that can be a separate PR